templates/bond_Debian.j2

```diff
@@ -1,6 +1,9 @@
 # {{ ansible_managed }}
 
 auto {{ item.device }}
+{% if item.bootproto == 'dhcp' %}
+iface {{ item.device }} inet dhcp
+{% endif %}
 {% if item.bootproto == 'static' %}
 iface {{ item.device }} inet static
 {% if item.pre_up is defined %}
```

Comment:
This is still only for static.

Reply:
Same for MTU

Reply:
I'm not sure which parameter are you referring to, but MTU is in the conditional code block with static. You can see it here https://github.com/michaelrigart/ansible-role-interfaces/pull/102/files/33edf8a6c9978ac05b700bf34c091541d4eb9244#diff-726051e02ea10b8f7d9545d5ca44548f287dff190e0ade05ba61989809f6e919R7-R24

Reply:
You're right, but it should not be. One benefit of your change is that it should remove inconsistencies like that.

Reply:
Of course, MTU is interface-related, not DHCP related. I overlooked that issue. Is there any similar parameter, which should be moved outside of static?

Reply:
Fixed issue for pre-up & mtu